Makefile

```diff
@@ -26,7 +26,8 @@ test:
 
 .PHONY: test-with-localnet
 test-with-localnet:
-	cargo nextest run -p sui-graphql-client
+	$(MAKE) -C crates/sui-transaction-builder test-setup
+	cargo nextest run -p sui-graphql-client -p sui-transaction-builder
```

Comment:
only change would be that I think `.PHONY` would need to be removed from `test-with-localnet` now since it actually does have a dependency based on files on disk

Reply:
To me `.PHONY` means "always run this rule, regardless of whether its rules are up-to-date or not" (which we want because we do always want to re-run the tests), but specifying the dependencies will ensure that if the JSON files are out-of-date w.r.t. to their respective inputs, they will be regenerated.

Having said that though, I now realise that we need to properly specify the dependencies in the `package_%.json` rule -- it needs to depend on everything inside the relevant test fixture folder.

Reply:
I think I fixed this as well.